### PR TITLE
Added maintainer docs for Azure

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
This can only be merged after https://github.com/conda-forge/conda-smithy/pull/1043 is merged

File: src/maintainer/infrastructure.rst [hunk 1]
```diff
@@ -111,17 +111,58 @@ Entering the above phrase in an issue of a feedstock will update the Circle-CI S
...
+Restarting builds
+.................
+
+Presently azure does not sync githuhb users. In order to restart a build you can restart it from the github checks interface.
+If that doesn't work, a close/open will kick off a new build.
+
+Using azure for *everything*
+............................
+
+Azure is the default provider for Linux and OS X.  To use azure for everything dd the following to ``conda-forge.yml`` in the root
```
Comment (croth1) [inline src/maintainer/infrastructure.rst]: `dd` -> `add`

File: src/maintainer/infrastructure.rst [hunk 2]
```diff
@@ -111,17 +111,58 @@ Entering the above phrase in an issue of a feedstock will update the Circle-CI S
...
+of the feedstock.
+
+.. code-block:: yaml
+
+    provider:
+      win: azure
+
+Presently azure has some issues building libraries using cmake on windows.  Azure does not have a VS2008 installation so building
```
Comment (croth1) [inline src/maintainer/infrastructure.rst]: Can we do this in a `.. note::` directive for consistency?

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**